src/lavinmq/amqp/queue/queue.cr

```diff
@@ -802,6 +802,7 @@ module LavinMQ::AMQP
         expire_msg(sp, :rejected)
       end
     rescue ex : MessageStore::Error
+      @log.error(ex) { "Queue closed due to error" }
       close
       raise ex
```

Comment:
where will this be reraised to? Should we log there instead?

Reply:
I put it here so it would be more centralized, so we dont have to change the log string in multiple places if we make changes.. but maybe we want to have different logs at different places where we do `reject`?